### DISCUSSION TITLE
Adding onClick handler to particular elements inside lottie

### DISCUSSION THREAD

**Discussion by rawalabhinav** [lottie-react-native/lottie-react-native#1224] [Q&A] [answered]
**Use Case**
I have a lottie animation, within which there's an element upon click on which I want to switch to another screen(by resolving a deep link).

**Problem**
How to access and add event listeners to a particular object inside lottie using `LottieView`?

**Hack**
The only way I found to do it is have a `TouchableOpacity` overlay above the lottie for the particular object, luckily that object in my lottie is static.

**Comment by TheRogue76** [comment, marked as answer]
Lottie animations are basically drawings, they are not separate elements to be accessed. Your approach with Touchable opcacity is probably the way to go
